From 90a096fc6a534fa8a70c1ff05742a384308b5971 Mon Sep 17 00:00:00 2001
From: Wenbin song <wenbin.song@nxp.com>
Date: Mon, 20 Nov 2017 16:22:47 +0800
Subject: [PATCH 203/211] armv8: ls1046aqds: update IFC NOR timings

Update IFC NOR timings to fix that the NOR flash can not
be erased with V4 FPGA image on ls1046aqds.

Signed-off-by: Wenbin Song <wenbin.song@nxp.com>
---
 include/configs/ls1046aqds.h |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/include/configs/ls1046aqds.h b/include/configs/ls1046aqds.h
index df70bd3..b619dc7 100644
--- a/include/configs/ls1046aqds.h
+++ b/include/configs/ls1046aqds.h
@@ -194,10 +194,10 @@ unsigned long get_board_ddr_clk(void);
 #define CONFIG_SYS_NOR_FTIM1		(FTIM1_NOR_TACO(0x35) | \
 					FTIM1_NOR_TRAD_NOR(0x1a) | \
 					FTIM1_NOR_TSEQRAD_NOR(0x13))
-#define CONFIG_SYS_NOR_FTIM2		(FTIM2_NOR_TCS(0x4) | \
-					FTIM2_NOR_TCH(0x4) | \
+#define CONFIG_SYS_NOR_FTIM2		(FTIM2_NOR_TCS(0x8) | \
+					FTIM2_NOR_TCH(0x8) | \
 					FTIM2_NOR_TWPH(0xe) | \
-					FTIM2_NOR_TWP(0x1c))
+					FTIM2_NOR_TWP(0x38))
 #define CONFIG_SYS_NOR_FTIM3		0
 
 #define CONFIG_SYS_MAX_FLASH_BANKS	2	/* number of banks */
-- 
1.7.1

